pkg/factory/factory.go

```diff
@@ -31,15 +31,12 @@ import (
 	cmclient "github.com/cert-manager/cert-manager/pkg/client/clientset/versioned"
 )
 
-var (
-	kubeConfigFlags = genericclioptions.NewConfigFlags(true)
```

Comment:
I made these global variables into local variables, so the test can parse flags multiple times without issues.